index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body class="container bod">
+    <header class="container navigation">
+      <h1 class="logo">Alex Hamn</h1>
```

Comment:
- [ ] [OPTIONAL] Would it be nice if we made the logo linked to your home page? 🤔 

Try to implement that using an `<a>` tag 👍🏼 

That would enhance your UX. 💹👥